bokehjs/src/less/crossfilter.less

```diff
@@ -13,11 +13,11 @@
 }
 
 .bk-crossfilter-panel-continuous-heading {
-    background-color: #396285 !important;
+ //   background-color: #396285 !important;
```

Comment:
If you intend to remove code, just do it and version control software will take care care of the rest. It's actually more readable if code that's no more needed is completely removed.